packages/scm/src/browser/scm-widget.tsx

```diff
@@ -222,8 +221,10 @@ export class ScmWidget extends ReactWidget implements StatefulWidget {
                 tabIndex={1}
                 value={input.value}
                 onChange={this.setInputValue}
-                ref={this.setInput}>
-            </textarea>
+                innerRef={((ref: HTMLTextAreaElement) => this.input = ref)}
```

Comment:
we should not use arrow functions in react callbacks, it breaks react element caching

Reply:
please use `this.setInput` instead